crates/bitwarden-wasm-internal/src/generators.rs

```diff
@@ -0,0 +1,56 @@
+use std::rc::Rc;
+
+use bitwarden_core::Client;
+use bitwarden_generators::{
+    GeneratorClientsExt, PassphraseGeneratorRequest, PasswordGeneratorRequest,
+};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct GeneratorClient(Rc<Client>);
+
+impl GeneratorClient {
+    /// Constructs a new SDK client for generating random passwords and passphrases
+    ///
+    /// # Arguments
+    /// - `client` - The client used to access the SDK
+    ///
+    /// # Returns
+    /// - `Self` - Returns newly constructed client
+    pub fn new(client: Rc<Client>) -> Self {
+        Self(client)
+    }
+}
+
+#[wasm_bindgen]
+impl GeneratorClient {
+    /// Generates a password from a provided request
+    ///
+    /// # Arguments
+    /// - `request` - Settings for the character sets and password length
+    ///
+    /// # Returns
+    /// - `Ok(String)` containing the generated password
+    /// - `Err(PasswordError)` if password generation fails
+    pub fn generate_password(
```

Comment:
We should try to keep the WASM API and the Rust API consistent on naming as much as we can, in the future I want to combine both APIs into one to avoid having to have all these duplicated functions, and that's going to be more complicated if both APIs diverge significantly.

So both functions should have the same name, either `generate_password` or `password` is fine by me, as long as they are consisten. Same thing would apply to passphrase.